articles/azure/asz-sco

```diff
@@ -0,0 +1,262 @@
+
+In line with UKCloud's SISP, we provide notification of customer-impacting security incidents. It is your responsibility to report similar incidents to us.
+
+## Platform management
+
+Users can access, manage and view the UKCloud for Microsoft Azure service, accessing only those features allowed by their role, in any of the following ways:
+
+- **Azure Stack API.** Enables the programmatic creation and management of the services inside the platform.
+
+- **Azure Stack graphical UI.** Provides a graphical interface to access the Azure Stack environment (depending on assigned permissions).
+
+- **UKCloud Portal.** Access to service reporting and incident and request management.
+
+You cannot access the underlying infrastructure. This includes (but isn't limited to) the hardware.
+
+## Service reporting
+
+**Visibility.** Customers can utilise the Azure Stack portal to configure multiple views of their estate to monitor the health of their services in real time. Including utilisation of individual components, alerts and recent operations.
+
+**Reporting** We provide a monthly service report via the UKCloud Portal. This includes a summary of platform availability and a retrospective of the interactions with UKCloud support during the previous month. Maintenance notifications and Service Status reports are also delivered through the Portal and via the UKCloud status page https://status.ukcloud.com/.
+
+**Billing.** We provide you with monthly bills covering your monthly spend.
+
+## Customer service
+
+**Customer Success Manager (CSM).** All new customers of the service have access to a UKCloud Customer Success Manager, who will help you with your adoption of the UKCloud for Microsoft Azure service,  including finding relevant systems and using UKCloud tools and processes.
+
+**Cloud Architects.** UKCloud Cloud Architects support you during the design of solutions for the cloud platform. UKCloud Cloud Architects are ideally placed to help reconcile your requirements with the UKCloud platforms. We recommend engagement with a Cloud Architect when implementing complex solutions, such as those using HybridConnect or a Walled Garden.
+
+**Technical Account Managers (TAM).** You will be allocated with an assigned point of contact who will provide ongoing assistance with reporting and incident escalation.
+
+**Support.** After the initial on-boarding and design phase, you can utilise the standard UKCloud support entitlement, which is documented in the [*Customer Engagement Factsheet*](https://ukcloud.com/wp-content/uploads/2017/07/UKCloud-Factsheet-Customer-Care.pdf).
+
+## Customer responsibilities
+
+- You are responsible for defining which Azure Active Directory (AAD) you wish to link their UKCloud for Microsoft Azure service too. The customer can choose to link to their existing AAD, or ask UKCloud to create a new AAD solely for the use of this service.
```

Comment:
You are responsible for defining which Azure Active Directory (AAD) you want to link your UKCloud for Microsoft Azure service to. You can choose to link to your existing AAD, or ask UKCloud to create a new AAD solely for the use of this service.